charts/verdaccio/templates/deployment.yaml

```diff
@@ -37,7 +37,7 @@ spec:
       {{- end }}
       containers:
         - name: {{ template "verdaccio.name" . }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

Comment:
Is there an space ? (Seeing from mobile)

Reply:
There are spaces, yes

<img width="805" alt="image" src="https://user-images.githubusercontent.com/4052340/163959191-39901eeb-355f-4dba-bfcd-a23e9c3ddd83.png">

Reply:
Ahh ok :) intended default and chart are different things